Python/generated_cases.c.h

```diff
@@ -436,10 +436,10 @@
         }
 
         TARGET(BINARY_SUBSCR_GETITEM) {
-            uint32_t type_version = read_u32(next_instr + 1);
-            uint16_t func_version = read_u16(next_instr + 3);
             PyObject *sub = PEEK(1);
             PyObject *container = PEEK(2);
+            uint32_t type_version = read_u32(next_instr + 1);
+            uint16_t func_version = read_u16(next_instr + 3);
```

Comment:
What happened here is that the relative order in which stack and cache effects are generated is reversed, due to the split of "stuff that's the same for macros and regular instructions" (this includes cache effects) from "stuff that's different" (mainly stack effects).

Other than this there are no changes in the generated code, because there are no macros with cache effects in the input yet. That will change in GH-99764 though.